wp-includes/sqlite/class-wp-sqlite-translator.php

```diff
@@ -2424,7 +2424,7 @@ private function strip_sqlite_system_tables( $tables ) {
 			array_filter(
 				$tables,
 				function ( $table ) {
-					$table_name = property_exists( $table, 'Name' ) ? $table->Name : $table->table_name; // phpcs:ignore WordPress.NamingConventions.ValidVariableName.UsedPropertyNotSnakeCase
+					$table_name = is_object( $table ) && property_exists( $table, 'Name' ) ? $table->Name : $table->table_name; // phpcs:ignore WordPress.NamingConventions.ValidVariableName.UsedPropertyNotSnakeCase
```

Comment:
Hmmm I don't think this would actually fix the issue...
if it's not an object, it will try to get `$table->table_name` which will again error.

Perhaps we should just add an additional condition _above_ this line?
Something like this:
```php
if ( ! is_object( $table ) ) {
	return false;
}
```